### ISSUE TITLE
not scraping all group members

### ISSUE THREAD

**Issue by Soyeb78692** [redianmarku/whatsapp-phone-number-scraper#1]
I tried and found its only scraping 5 to 10 members of the group

**Comment by binyamindavid** [comment]
you are right

**Comment by mohammednabarawy** [comment]
fixed with new scraping method

**Comment by Karytonn** [comment]
Manually scroll and solve this problem 😉

**Comment by RobertNerd** [comment]
True! Actually mine downloads nothing. Like it downloads and the file is empty, no contacts. Kindly, check on that.

**Comment by chingyawhao** [comment]
Whatsapp contact, that page is implemented as infinite scroll meaning it will load additional contacts when it scroll
To solve that for large groups, check https://github.com/redianmarku/whatsapp-phone-number-scraper/pull/5

Also if you have the contact saved, then it doesn't show the phone number and it won't be saved in the list.
